ports/lose95/README.md

```diff
@@ -0,0 +1,16 @@
+## Notes
+
+A special thank you to [Kultisti](https://kultisti.itch.io/) for making this great PC GUI action game and allowing us to distribute it Ready to run !
```

Comment:
Missing URL to the game

Reply:
Added link to the Lose95 game on Itch instead of the creator's Itch page.